src/aks-preview/azext_aks_preview/decorator.py

```diff
@@ -2345,61 +2345,63 @@ def check_raw_parameters(self):
         )
 
         if not is_changed and is_default:
-            # Note: Uncomment the followings to automatically generate the error message.
```

Comment:
I dont know where this `check_raw_parameters` call is used, so approving but probably confirm with somebody who knows about this to make sure it's good (seems fine to me but what do I know).

Reply:
@FumingZhang let usk now if if there is anything else strange about doing htis here but bascically we want no arguments to be a prompt not a failure now.

Reply:
Yeah, the changes look good to me. Do we want to introduce this behavior to official cli? I mean allowing the put request even if no argument specified.

Reply:
I think our prefernce is to let it bake in the extension for a month and get some feedback from support and field and then put it into official after we get faceback.